src/components/App.js

```diff
@@ -1,7 +1,48 @@
-import React from 'react'
+import React, { useState } from 'react'
+import Button from 'components/Button/Button'
+import Container from 'components/Container/Container'
+import Input from 'components/Input/Input'
+import List from 'components/List/List'
+import ListItem from 'components/ListItem/ListItem'
 
 const App = () => {
-  return <></>
+  const [items, setItems] = useState([])
+  const [todo, setTodo] = useState('')
+
+  const addItemList = (event) => {
```

Comment:
Siguiendo la convención del issue #6 , es mejor cambiar el nombre del handler por `handleKeyDown` o, mucho más específico, ya que luego vamos a tener otros inputs y tener que cambiarlo:

```suggestion
  const handleInputTodoKeyDown = (event) => {
```